rasa/shared/importers/importer.py

```diff
@@ -48,16 +48,23 @@ async def get_stories(
         Returns:
             `StoryGraph` containing all loaded stories.
         """
-
+        # TODO: Drop `use_e2e` in Rasa Open Source 3.0.0 when removing Markdown support
```

Comment:
This is confusing now that `use_e2e` also determines which files are loaded right?

Reply:
I hope the renamed parameters make this less confusing. The importer only uses `use_e2e` for configuring the markdown reading. Means we no longer need it when Markdown is removed.